app/views/drawings/index.html.haml

```diff
@@ -1,5 +1,2 @@
 #drawings
   = render 'drawings'
-
-#paginator.text-center
-  = link_to_next_page @drawings, 'MORE', remote: true, id: 'load_more'
```

Comment:
Ah ok! Yep it's a bit buggy so need to fix it and remove the purple blob altogether. We use the [Kaminari gem]( https://github.com/amatsuda/kaminari) for pagination which hooks up some handy methods like `link_to_next_page`, but in terms of how that method is called is all JS in the partial. So if we can update the JS to call the method on scroll then yay!